src/pages/home/report/ReportActionItem.js

```diff
@@ -27,6 +29,16 @@ const propTypes = {
 
     // Should we display the new indicator on top of the comment?
     shouldDisplayNewIndicator: PropTypes.bool.isRequired,
+
+    // Information about the network
+    network: PropTypes.shape({
+        // Is the network currently offline or not
```

Comment:
NAB: But the property name `isOffline` is pretty self-explanatory without the doc line.

Reply:
I borrow the idea from Style guide. So I think we should keep it.

Reply:
Yep, `propTypes` should always have doc lines